Analytics/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.7.0</string>
+	<string>3.8.0-beta.0</string>
```

Comment:
This should be 3.8.0 (due to https://github.com/segmentio/analytics-ios/issues/794).